main.c++

```diff
@@ -0,0 +1,77 @@
+#include<iostream>
+using namespace std;
+
+class Room{
+    private:
+    string room_type;
+    int capacity;
+
+    public:
+    Room(string room_type,int capacity){
+        if(!(room_type=="classroom"||room_type=="lecture_hall")){
+            throw "Invalid Room Type";
```

Comment:
This is not a recommended practice. It's better to use standard exceptions like `std::invalid_argument` or `std::runtime_error`.

Reply:
Done 👍